test/error-placement.js

```diff
@@ -291,6 +291,42 @@ test( "test existing non-label used as error element", function(assert) {
 	assert.noErrorFor( field );
 });
 
+test( "test aria-describedby with input names contains special characters (:, ., [, ], $)", function() {
```

Comment:
I think all of these meta-characters will result in the problem:
```
!"#$%&'()*+,./:;<=>?@[\]^`{|}~
```

mentioned here: https://api.jquery.com/category/selectors/